### DISCUSSION TITLE
头文件报错，QT自带的函数也报错

### DISCUSSION THREAD

**Discussion by ling9535464** [liusy58/NJU_AdvancedProgramming_SP22#174] [Q&A]
![屏幕截图 2022-04-18 200525](https://user-images.githubusercontent.com/103937881/163805917-8f74e519-7bfc-4132-9d02-cc4dc9ac90cd.png)
![屏幕截图 2022-04-18 200451](https://user-images.githubusercontent.com/103937881/163805925-f8a6f439-07c0-4491-8dc0-63ba9ad74cd4.png)
![屏幕截图 2022-04-18 200512](https://user-images.githubusercontent.com/103937881/163805932-7b9ec7c3-c29b-405f-81d8-5b1f9871131f.png)

**Comment by liusy58** [comment]
先看看这个  #169
